topsort-api-v2.yml

```diff
@@ -1141,6 +1142,11 @@ components:
           exclusiveMinimum: true
           description: The price of a single item in the marketplace currency.
           example: 12.95
+        vendorId:
```

Comment:
I think we also need to explain when to add vendorId.

Reply:
Agreed, we probably should mention that this is generally optional and it is required in case they have multiple providers or when they want to use halo attribution.

Reply:
Added that